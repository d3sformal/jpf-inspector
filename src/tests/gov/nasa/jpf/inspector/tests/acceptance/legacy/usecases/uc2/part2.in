cr bp pos = *Crossing.java:90
run
step_out
print #thread[1].#stackFrame[0]
so
so
si
print #thread[0].#stackFrame[0]
so
bso
bso
run
assert *Crossing.java:217 finished == false
step_instruction
bsins
print #thread[0].#stackFrame[5]
print #thread[0]
cr bp choice_generator = data
cr bp position = *Crossing.java:194
continue
print #thread[0].#stackFrame[1]
print #thread[0].#stackFrame[0]
print #thread[0].#stackFrame[0].p2
print #thread[0].#stackFrame[0].isNative
bsout
run
sout
sins
wait