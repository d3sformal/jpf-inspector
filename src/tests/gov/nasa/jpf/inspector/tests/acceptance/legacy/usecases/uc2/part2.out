INFO: JPF created and connected, SuT is started
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=1 position=*Crossing.java:90]
	SuT will now execute "getstatic gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc2.Bridge.onBridge" at position *Crossing.java:90.
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc2/Crossing.java:200 (iadd), source: total += Bridge.Cross();
ERR: There is no thread with the index 1.
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc2/Crossing.java:202 (iload_1), source: if (isNative) {
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc2/Crossing.java:205 (iload_2), source: Verify.ignoreIf(total > 17); //with this DFS will also find error
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/vm/Verify.java:-1 (executenative JPF_gov_nasa_jpf_vm_Verify.ignoreIf__Z__V), source unavailable
gov.nasa.jpf.vm.Verify.ignoreIf - (in file gov/nasa/jpf/vm/Verify.java:-1)
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc2/Crossing.java:208 (invokestatic gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc2.Bridge.clearBridge()V), source: Bridge.clearBridge();
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc2/Crossing.java:205 (iload_2), source: Verify.ignoreIf(total > 17); //with this DFS will also find error
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc2/Crossing.java:202 (iload_1), source: if (isNative) {
INFO: Breakpoint hit: [1 :  state=ENABLED hits=2 hitsTotal=2 position=*Crossing.java:90]
	SuT will now execute "getstatic gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc2.Bridge.onBridge" at position *Crossing.java:90.
New assertion successfully created with ID 9.
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc2/Crossing.java:90 (iconst_0), source: if (onBridge[0].time > onBridge[1].time) {
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc2/Crossing.java:90 (getstatic gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc2.Bridge.onBridge), source: if (onBridge[0].time > onBridge[1].time) {
ERR: No stack frame at depth 5 in thread 0.
0 : main state=RUNNING priority=5
	0 : gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc2.Bridge.Cross - (in file gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc2/Crossing.java:90) - if (onBridge[0].time > onBridge[1].time) {
	1 : gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc2.Crossing.main - (in file gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc2/Crossing.java:200) - total += Bridge.Cross();
	2 : gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc2.Crossing.[main] - (in file gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc2/Crossing.java:0)
INFO: Breakpoint hit: [13 :  state=ENABLED hits=1 hitsTotal=1 position=*Crossing.java:194]
	SuT will now execute "aload 4" at position *Crossing.java:194.
gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc2.Crossing.[main] - (in file gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc2/Crossing.java:0)
Stack slots
	0 : ???-RawView-no Name or Type provided (int) = 187
gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc2.Crossing.main - (in file gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc2/Crossing.java:194) - p1.move();
Stack slots
	0 : args (java.lang.String[]) = []
	1 : isNative (boolean) = false
	2 : total (int) = 4
	3 : finished (boolean) = false
	4 : p1 (gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc2.Person) = gov.nasa.jpf.inspector.tests.acceptance.legacy....
	5 : p2 (gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc2.Person) = gov.nasa.jpf.inspector.tests.acceptance.legacy....
	6 : p3 (gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc2.Person) = gov.nasa.jpf.inspector.tests.acceptance.legacy....
	7 : p4 (gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc2.Person) = gov.nasa.jpf.inspector.tests.acceptance.legacy....
p2 (gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc2.Person) = gov.nasa.jpf.inspector.tests.acceptance.legacy....
  Instance fields
	0 : id (int) = 2
	1 : time (int) = 2
	2 : side (boolean) = true
	1 : isNative (boolean) = false
ERR: Backwards step not possible (there is no appropriate step left for this thread to backtrack to).
INFO: Breakpoint hit: [12 :  state=ENABLED hits=1 hitsTotal=1 choice_generator=data]
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc2/Crossing.java:150 (ifne 21), source: if (!Verify.getBoolean()) {
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc2/Crossing.java:151 (aload_0), source: Bridge.tryToCross(this);
INFO: JPF Terminating